spec/latest/json-ld-api/index.html

```diff
@@ -3306,8 +3345,6 @@
                 <li>While <em>property</em> equals <code>rdf:rest</code>,
                   the <a>array</a> value of the member of <em>node usages map</em> associated with the <code>@id</code>
                   member of <code>node</code> has only one member,
-                  the value associated to the <code>usages</code> member of <em>node</em> has
-                  exactly 1 entry,
```

Comment:
While I would like to revisit these algorithms with an eye towards removing implementation details and focusing on the intent of the algorithms, this seems to have been an accidental change, which I'll reverse.